frontend-web/webclient/app/UCloud/ProvidersApi.tsx

```diff
@@ -204,6 +274,19 @@ class ProviderApi extends ResourceApi<Provider, Product, ProviderSpecification,
         />;
     };
 
+    update(
+        request: ProvidersUpdateSpecificationRequest
+    ): APICallParameters<ProvidersUpdateSpecificationRequest, any /* unknown */> {
+        return {
+            context: "",
+            method: "POST",
+            path: "/api/providers" + "/update",
+            parameters: request,
+            reloadId: Math.random(),
+            payload: request,
+        };
+    }
```

Comment:
@DanThrane, I haven't used it much, but `apiUpdate` should replace the object created here, right?